scipy/optimize/_linprog_highs.py

```diff
@@ -89,36 +100,33 @@ def _linprog_highs(lp, solver, time_limit=None, presolve=True,
     lp :  _LPProblem
         A ``scipy.optimize._linprog_util._LPProblem`` ``namedtuple``.
     solver : "ipm" or "simplex" or None
-        Which HiGHS solver to use.  If `None`, HiGHS will determine which
+        Which HiGHS solver to use.  If ``None``, HiGHS will determine which
         solver to use based on the problem.
 
     Options
     -------
     maxiter : int
-        The maximum number of iterations to perform in either phase.
+        The maximum number of iterations to perform in either phase. For
+        ``solver='ipm'``, this does not include the number of crossover
+        iterations.
     disp : bool
         Set to ``True`` if indicators of optimization status are to be printed
         to the console each iteration; default ``False``.
     time_limit : float
         The maximum time in seconds allotted to solve the problem; default is
         unlimited.
     presolve : bool
-        Set to ``False`` if presolve is to be disabled; default ``True``.
-    autoscale : bool
-        Set to ``True`` to automatically perform equilibration.
-        Consider using this option if the numerical values in the
-        constraints are separated by several orders of magnitude.
-        Default: ``False``.
-    unknown_options : dict
-        Optional arguments not used by this particular solver. If
-        `unknown_options` is non-empty, a warning is issued listing all
-        unused options.
+        Presolve attempts to  identify trivial infeasibilities,
+        identify trivial unboundedness, and simplify the problem before
+        sending the problem to the main solver. It is generally recommended
+        to keep the default setting ``True``; set to ``False`` if presolve is
+        to be disabled.
     dual_feasibility_tolerance : double
         Dual feasibility tolerance
     dual_objective_value_upper_bound : double
         Upper bound on objective value for dual simplex:
         algorithm terminates if reached
-    message_level : int {0, 1, 2, 4}
+    message_level : int {0, 1, 2, 4, 7}
         Verbosity level, corresponds to:
 
             ``0``: ML_NONE
```

Comment:
Are human-readable values available for these?

Reply:
@mdhaber Yes, I can export those in the same way as I did the status codes.  I'll make a PR to that effect